README.md

```diff
@@ -1 +1,242 @@
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> key features of the application.
+
+- **HTML heading**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> link to deployed project.
+> N/A
+
+<!-- - [Live Demo Link](https://google.com) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To make use of this project.
+
+Clone this GitHub repo and run it locally on your machine.
+
+### Prerequisites
+
+In order to run this project you need:
+
+**a web browser**
+
+### Setup
+
+Clone this repository to your desired folder (ex: my-folder):
+
+```sh
+  cd my-folder
+  git clone [git@github.com:myaccount/my-project.git](https://github.com/Paulinelali/module-1-bk-1-hello-microverse.git)
+```
+--->
+
+### Install
+
+<!-- Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+---> -->
+
+### Usage
+
+To run the project, execute the following command:
+
+Open the index.html file in your browser
+
+<!-- ### Run tests
+
+To run tests, run the following command: -->
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+<!-- You can deploy this project using: -->
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Collaborator of this project.
+
+👤 **Author1**
+
+- GitHub: [@Paulinelali](https://github.com/Paulinelali)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ayomide-emmanuel-1353b31b9/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Contact form**
+- [ ] **user behavious**
+- [ ] **animations**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project...
+Kindly follow me on GitHub(https://github.com/Paulinelali)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] The previous reviewer only asked you to remove this text - `Give credit to everyone who inspired your codebase.` and not the Entire Acknowledgements section. Kindly add that section back without the text `Give credit to everyone who inspired your codebase.`. Then update this text `I would like to thank...` with your acknowledgement to whom ever you choose. 


![image](https://user-images.githubusercontent.com/84156714/235029578-b4f3f644-fb33-486f-b3f4-0cef8a4617ff.png)

Reply:
Is it compulsory to have Acknowledgments? I choose nobody, no one. What's the problem with that?

Reply:
NOBODY INSPIRED MY CODE!